src/AppBundle/Controller/CommitteeController.php

```diff
@@ -143,10 +148,37 @@ public function listMembersAction(Committee $committee): Response
     {
         $committeeManager = $this->get('app.committee_manager');
 
+        $members = $this->getDoctrine()->getRepository(CommitteeMembership::class)->findMembers($committee->getUuid());
```

Comment:
Why not having a `findCommitteeMembers()` method to the `CommitteeManager` service class instead. It makes more sense to me.